sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IO.java

```diff
@@ -260,15 +266,12 @@ public static class Read extends PTransform<PCollection<String>, Read.Result> {
 
     public Read() {}
 
-    public static class Result implements POutput, PInput {
-
-      private PCollection<HL7v2Message> messages;
-
-      private PCollection<HealthcareIOError<String>> failedReads;
-      PCollectionTuple pct;
+    public static class Result extends HL7v2ReadResult<String, HL7v2Message> {
```

Comment:
ideally each public class (that would be rendered to javadoc) should have docstring. Would you mind add a brief docstring also here?

Reply:
Thanks for mentioning, added  brief docstring.